pages/design/process.md

```diff
@@ -204,6 +204,17 @@ Published rules
   - went through Call for Review (if applicable), and change requests have been resolved
   - been merged and published as a proposed rule in the WAI website at https://www.w3.org/WAI/standards-guidelines/act/rules/ (ACT Rules on the WAI website are generally updated every Tuesday)
 
+### Merge and publication policy
+
+To give implementors of ACT Rules time to update implementation data, pull requests cannot be merged on Friday, Saturday, or Sunday. Those three days are reserved for implementors to update their implementation data (automated or otherwise). The publication cycle works as follows:
```

Comment:
What happens if something gets merged on Monday, or on Tuesday, right before the WAI website update goes live? As I said yesterday, I think we should restrict the updates coming from the act-rules.github.io repo to a specific date and time during the week, which could be Thursdays after the meeting when hopefully all has been merged.